package-lock.json

```diff
@@ -2917,14 +2917,6 @@
 			"resolved": "https://registry.npmjs.org/ms/-/ms-2.1.3.tgz",
 			"integrity": "sha512-6FlzubTLZG3J2a/NVCAleEhjzq5oxgHyaCU9yYXvcLsvoVaHJq/s5xXI6/XXP6tz7R9xAOtHnSO/tXtF3WRTlA=="
 		},
-		"multiline": {
-			"version": "2.0.0",
-			"resolved": "https://registry.npmjs.org/multiline/-/multiline-2.0.0.tgz",
-			"integrity": "sha512-+HpXaUcV8PIGNNmuhtlaVmw4NH0W30/A5WP+rq6pxZYBjDslX/sXkFgL3Mgk1cSGGIICjWu4gNStkJXL6ZM2DQ==",
-			"requires": {
-				"strip-indent": "^2.0.0"
```

Comment:
I wonder why strip-indent is not removed as it is not used by some other package.
Could you please remove the "node_modules" folder and run "npm install" again (with npm v6.x)?